app/routers/vip.py

```
@@ -1,7 +1,19 @@
 import logging
 import os
 import inspect
-from typing import TYPE_CHECKING, Any, Callable, Dict, Literal, Mapping, Optional, Type, Union, cast
+from typing import (
```

Flake8 (7.3.0)

[error] 'typing.Annotated' imported but unused

(F401)